src/collections/_documentation/integrations/github.md

```diff
@@ -2,19 +2,56 @@
 title: GitHub
 sidebar_order: 2
 ---
+Sentry’s new Github integration has the following features: commit tracking, issue management, and Github SSO.You can now use the data from your commits to Github to help you find and fix bugs faster.
```

Comment:
Should you have a space after the `Github SSO.`?  GitHub also prefers to have the `H` capitalized. All their [press materials](https://github.com/logos) use big H.